modules/templates/ref-creating-templates.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-creating-templates_{context}"]
+= Creating templates
+
+You can create a template by defining a `Template` object as a YAML file.
+
+The `Template` object describes the template and its metadata. It also contains required input variables and a list of actions which are executed by the scaffolding service.
+
+.`Template` object example
+[source,yaml]
+----
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: template-name # <1>
+  title: Example template # <2>
+  description: An example template for v1beta3 scaffolder. # <3>
+spec:
+  owner: backstage/techdocs-core # <4>
+  type: service # <5>
+  parameters: # <6>
+      # First set of steps
+    - title: Fill in some steps
+      required:
+        - name
+      properties:
+        name:
+          title: Name
+          type: string
+          description: Unique name of the component
+        owner:
+          title: Owner
+          type: string
+          description: Owner of the component
+    # Second set of steps
+    - title: Choose a location
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Repository Location
+          type: string
+  steps: # <7>
+    - id: fetch-base
+      name: Fetch Base
+      action: fetch:template
+      # ...
+  output: # <8>
+    links:
+      - title: Repository # <9>
+      - title: Open in catalog # <10>
+# ...
+----
+<1> Specify a name for the template.
+<2> Specify a title for the template. This is the title that is visible on the template tile in the *Create...* view.
+<3> Specify a description for the template. This is the description that is visible on the template tile in the *Create...* view.
+<4> Specify the ownership of the service or component being created using the template. The `owner` field provides information about who is responsible for maintaining or overseeing the component within the system or organization.
++
+In the provided example, the `owner` field is set to `backstage/techdocs-core`. This means that components created by using this template belong to the `techdocs-core` project in the `backstage` namespace.
++
+--
+[TIP]
+====
+You can allow users to choose an owner for components that are created by using the link:https://backstage.io/docs/features/software-templates/writing-templates#the-owner-picker[`OwnerPicker` custom field].
+====
+--
+<5> Specify the component type. Common possible values for this field are the following:
```

Comment:
Do we want to call out that this could accept any value?  Like in the upstream docs here:
https://backstage.io/docs/features/software-catalog/descriptor-format/#spectype-required

Reply:
Updated the description for the next commit - hopefully that's clearer. Thanks!